corehq/apps/export/models/new.py

```diff
@@ -120,7 +119,7 @@ class ExportColumn(DocumentSchema):
     # Determines whether or not to show the column in the UI Config without clicking advanced
     is_advanced = BooleanProperty(default=False)
     selected = BooleanProperty(default=False)
-    tags = ListProperty()
+    tags = ListProperty()  # TODO: I think this is moving to SystemExportColumn?
```

Comment:
seems right to me. don't think we ever need tags on the regular export column. though could make it easy to just assume every column has a tags field so we do less `if tags then loop over them and do this`